samples/bluetooth/mesh/sensor_server/README.rst

```diff
@@ -0,0 +1,186 @@
+.. _bluetooth_mesh_sensor_server:
+
+Bluetooth: Mesh sensor server
+###############################
+
+The Bluetooth Mesh sensor server sample demonstrates how to set up a basic Mesh sensor server model application that provides sensor data to one :ref:`Mesh sensor client <bt_mesh_sensor_cli_readme>` model.
+
+.. note::
+    This sample must be paired with :ref:`bluetooth_mesh_sensor_client` to show any functionality. The server provides the sensor data used by the client.
+
+Overview
+********
+
+This sample is split into the following source files:
+
+* A :file:`main.c` file to handle initialization.
+* One additional file for handling Mesh models, :file:`model_handler.c`.
+
+The following Mesh sensor types are used in this sample:
```

Comment:
Could you provide a line of explanation after this list or in line 7 (which would be better, I think) about why these various sensor types are used in this sample?

Reply:
For example "Four different sensor types are used to showcase different ways for the server to publish data. In addition, usage of both single-channel sensors types and series sensor types are shown (reference: http://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/include/bluetooth/mesh/sensor.html?#bt-mesh-sensor-types-channels)."